internal/runner/runner.go

```diff
@@ -27,6 +28,19 @@ func NewRunner(options *Options) (*Runner, error) {
 
 // Run the instance
 func (r *Runner) Run() error {
+	// add default path to $PATH
+	if r.options.SetPath || r.options.Path == defaultPath {
+		if err := path.SetENV(r.options.Path); err != nil {
+			return errorutil.NewWithErr(err).Msgf(`Failed to set path: %s. Add %s to $PATH and run again`)
+		}
+	}
+
```

Comment:
while this logic is good . I think we can add some comments to point out that we modify path by default (I think it is  implied but not directly mentioned but would help to understand code if it was explicitly mentioned)

Reply:
I've updated the readme